translations/messages-en.properties

```diff
@@ -906,4 +906,7 @@ forms.json = JSON
 upload.xform.xml.help = The xform definition in XML.
 upload.xform.meta.help = The meta file in JSON format including the icon, title, context, and so on.
 password.weak = The password is too easy to guess. Include a range of types of characters to increase the score.
-password.length.minimum = The password must be at least {{minimum}} characters long.
\ No newline at end of file
+password.length.minimum = The password must be at least {{minimum}} characters long.
```

Comment:
I agree. Unfortunately the poeditor service exports these files without the trailing new line, so if we add it then POE keeps removing it anyway.